bootstrap/bin/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/local/bin/bash
+
+export BUILD_HARNESS_ORG=${1:-plus3it}
```

Comment:
Can maybe change the env namespace from BUILD_HARNESS to TARDIGRADE_CI?

Reply:
updated